Makefile

```
@@ -28,7 +28,7 @@ test: prepare-test ## Run the tests
 					sh -c "npm install tap-xunit -g && cat target/results.tap | tap-xunit --package='co.elastic.opbeans' > target/junit-results.xml"
 
 publish: build ## Publish docker image
-	@docker push opbeans/opbeans-go:${VERSION}
+	@echo docker push ${IMAGE}:${VERSION}
```

For testing purposes only
```suggestion
	@docker push ${IMAGE}:${VERSION}
```